src/main/webapp/app/core/about-us/about-us.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { ProfileService } from 'app/shared/layouts/profiles/profile.service';
+import { filter, tap } from 'rxjs/operators';
+import { ProfileInfo } from 'app/shared/layouts/profiles/profile-info.model';
+import { SERVER_API_URL } from 'app/app.constants';
+import { StaticContentService } from 'app/shared/service/static-content.service';
+import { AboutUsModel } from 'app/core/about-us/models/about-us-model';
+
+@Component({
+    selector: 'jhi-about-us',
+    templateUrl: './about-us.component.html',
+    styles: [],
+})
+export class AboutUsComponent implements OnInit {
+    private readonly issueBaseUrl = 'https://github.com/ls1intum/Artemis/issues/new?projects=ls1intum/1';
+    readonly bugReportUrl = `${this.issueBaseUrl}&labels=bug&template=bug-report.md`;
+    readonly featureRequestUrl = `${this.issueBaseUrl}&labels=feature&template=feature-request.md`;
+    email: string;
+
+    data: AboutUsModel;
+
+    readonly SERVER_API_URL = SERVER_API_URL;
+
+    constructor(private route: ActivatedRoute, private profileService: ProfileService, private staticContentService: StaticContentService) {}
+
+    /**
+     * On init get the json file from the Artemis server and save it.
+     * On init get the mail data needed for the contact
+     */
+    ngOnInit(): void {
+        this.staticContentService.getStaticJsonFromArtemisServer('/about-us.json').subscribe((data) => {
+            this.data = data;
+        });
+
+        this.profileService
+            .getProfileInfo()
+            .pipe(
+                filter(Boolean),
```

Comment:
Just trying to learn more about rxjs? Why are you doing this here? Is it possible that getProfileInfo sends undefined?

Reply:
Small explanation. This implementation here, together with the method set contact() are not done by me. They were implemented on footer component, but since Report a bug, Request a feature and Contact us are now inside the about page, I had to take this logic with in this component. I didnt overanalyze this part and gave the benefit of the doubt that when they were implemented back then, they have already been checked. That would be an answer also to the previous 2 comments.

As for rxjs question: getProfileInfo() here is a BehaviourSubject, an Observable used to stream data. We can subscribe to it and it always delivers immediately the latest data when data changes. The good of it is it will always send a valid value, even if no data has been emitted, thats why the risk of an error because of the undefined is eleminated. Thats why BehaviourSubjects are normaly initiated with undefined or null values (or empty values in general). It is actually also very good when you want to render the html parts only when data is there, not before, and can use BehaviourSubjetcs with async pipe to make sure of it.